src/errors.rs

```diff
@@ -38,3 +40,57 @@ pub enum PolicyEvaluatorError {
     #[error("protocol_version is only applicable to a Kubewarden policy")]
     InvokeWapcProtocolVersion(#[source] crate::runtimes::wapc::errors::WapcRuntimeError),
 }
+
+#[derive(Error, Debug)]
+pub enum PolicyEvaluatorBuilderError {
+    #[error("cannot convert given path to String")]
+    ConvertPath,
+
+    #[error("protocol_version is only applicable to a Kubewarden policy")]
+    InvokeWapcProtocolVersion(#[source] crate::runtimes::wapc::errors::WapcRuntimeError),
+
+    #[error(transparent)]
+    InvalidUserInput(#[from] InvalidUserInputError),
+
+    #[error("error when creating wasmtime engine: {0}")]
+    WasmtimeEngineBuild(#[source] anyhow::Error),
+
+    #[error("error when building wasm module: {0}")]
+    WasmModuleBuild(#[source] anyhow::Error),
```

Comment:
These should be converted from `anyhow::Error` to `wasmtime::Error`

Reply:
true! done and rebased.